pw.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+python "%~dp0pw" %*
```

Comment:
Can you elaborate on what this is and what it does?

Reply:
It executes the pw python script on windows and passes all arguments. This create a unified dev experience on windows and *nix.